app/models/short_url_request.rb

```diff
@@ -62,4 +69,8 @@ def strip_whitespace
     self.from_path = self.from_path.strip
     self.to_path = self.to_path.strip
   end
+
+  def should_republish_redirect?
+    accepted? && (from_path_changed? || to_path_changed?)
```

Comment:
I'm not sure this does the correct thing.  Because this is in an `after_update` hook, I think `from_path_changed?` etc will always be false.

Would be good to have unit tests covering this.